packages/components/mediaPlayer/MediaPlayerVideo.tsx

```diff
@@ -79,8 +79,10 @@ export const MediaPlayerVideo: FC<MediaPlayerVideoProps> = ({
   // Plug or not the playbackStatus from MediaPLayerProvider
   const statusToUse = useMemo(
     () => (isInMediaPlayer ? playbackStatus : localStatus),
-    [isInMediaPlayer, playbackStatus, localStatus],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
please let me understand why this is necessary?

Reply:
Do you mean the comment? The comments are just to disable the warning which is sent by react if we not set the variable that is used inside the useMemo.   The variables playbackStatus and localStatus  in the dependency array is continously changing which updates the statusToUse continously, as a result the video component is rendering again and again, which plays and pause the video automatically, which is the reason for our video not playing.

Reply:
@n0izn0iz seems like a valid reason."

Reply:
I recommend not to ignore the error and refactor to not trigger

Reply:
actually this memo and the one above should not be needed and are probably slowing down the render